schematic_api/api/routes.py

```diff
@@ -83,9 +75,25 @@ def export(self, spans: List[Span]) -> None:
                 f.write(span_json_one_line)
 
 
-trace.get_tracer_provider().add_span_processor(BatchSpanProcessor(OTLPSpanExporter()))
-# processor = SimpleSpanProcessor(FileSpanExporter("otel_spans_schemati_api.json"))
-# trace.get_tracer_provider().add_span_processor(processor)
+def set_up_tracing() -> None:
+    """Set up tracing for the API."""
+    tracing_export = os.environ.get("TRACING_EXPORT_FORMAT", None)
+    if tracing_export == "otlp":
+        trace.get_tracer_provider().add_span_processor(
+            BatchSpanProcessor(OTLPSpanExporter())
+        )
+        print("going through here")
```

Comment:
```suggestion
        print("Exporting trace data through OTLP Span Exporter")
```

Reply:
Thanks Bryan! I think that print statement should be removed.